pages/en/lb4/Using-components.md

```diff
@@ -7,24 +7,42 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Using-components.html
 summary:
 ---
-Components are an important part of LoopBack 4. We are keeping the core small and easy to extend, making it easy for independent developers to contribute additional features to LoopBack. Components are the vehicle for bringing more goodness to your applications.
 
-A typical LoopBack component is an [npm](https://www.npmjs.com/) package exporting a Component class which can be added to your application.
+One of the many ways LoopBack 4 allows for extensibility is through Components.
+A Component makes it easy for independent developers to contribute additional
+features to LoopBack. Components provide a package that allows you to extend
+your Application.
 
-```js
-import {Application} from '@loopback/core';
-import {AuthenticationComponent} from '@loopback/authentication';
+A typical LoopBack component is an [npm](https://www.npmjs.com) package
+exporting a Component class whih can be added to your application.
 
-const app = new Application();
+```ts
+import { RestApplication } from "@loopback/rest";
+import { AuthenticationComponent } from "@loopback/authentication";
+
+const app = new RestApplication();
+// Add component to Application, which provides bindings used to resolve
+// authenticated requests in a Sequence.
 app.component(AuthenticationComponent);
 ```
 
-In general, components can contribute the following items:
+Components can contribute the following items:
+
+* [Controllers](Controllers.html)
+* Providers of additional [Context values](Context.html)
 
- - [Controllers](Controllers.html)
- - Providers of additional [Context values](Context.html)
+LoopBack 4 was built with extensibility in mind and this includes Components,
+which can be allowed to contribute additional artifacts by adding a Mixin
+to your Application class. This doesn't change how a a Component is registered
+(`app.component()`) but it enables the Component to contribute additional artifacts.
+For example:
 
-In the future (before the GA release), components will be able to contribute additional items:
+* `RepositoryMixin` from `@loopback/repository` enables a Component to
+  contribute [Repositories](Repositories.html)
+* `BootMixin` from `@loopback/boot` enables a Component to
+  contribute [Booters](Booting-an-Application.html#booters)
 
- - Models
- - [Repositories](Repositories.html)
```

Comment:
I wonder why we start to remove models/repositories here.

Reply:
I've rephrased the above bullets on adding Repositories via a Component. We don't current have support for contributing Models via a Component AFAIK and I would like docs to reflect the current implementation.